llamaindex_k8s/index_store/summary2vector/5_doc_summary/index_store.json

```diff
@@ -0,0 +1 @@
905f2e36-68a2-4c59-96f2-499a41b7feac\", \"7237af19-a19c-461a-bdf2-d8853bbee804\"]}, \"node_id_to_summary_id\": {\"9f660d3d-c4fc-46b7-a164-9e706c2ecf5b\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"f29ea54b-3508-4efd-b2e5-25896fae2d7d\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"7ae71f8e-b2d9-48bd-a987-99b6e006c6fa\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"6be68af7-b364-4a1c-a68c-f2230810e994\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"a6f6cbf2-fed3-4e97-9701-f1caae72c84d\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"02008213-333f-4eb8-ad62-51bd12db550b\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"230e47e3-4347-4747-8fe3-deba5d34072a\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"6f48da75-9fae-47e9-a888-9294c2febae1\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"839a47d3-bc16-4659-8753-a4d36e4795e5\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"0906bd1e-3442-43c8-b644-1f40ab903345\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"a74c6ead-ef25-4c22-8941-03e9948ade92\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"132af512-2813-4af4-84b3-9ba2117411ad\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"7ef9ca80-749c-41e7-b235-65c981ffe19e\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"43935849-70a9-4818-a0c6-8ca1ebf67635\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"309c8a74-4c3b-46b7-90a9-a2c12b81a992\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"7f04c1b5-33c6-4e6f-8936-2eefc9b0e279\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"9c455931-3906-4fb5-90ee-eeed1eedeb46\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"711d103e-3861-400c-b0f8-3c904760f120\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"e4fdc969-6615-4814-afe6-8e9eedf35fe3\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"fde24d96-63bf-4189-9195-233753970ffb\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"0a208132-33b2-45b3-9d2f-2d622d4732f3\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"080c0bcc-5645-4cf4-94cc-3158f769c1b9\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"5cbd3422-8b0b-439d-9ae5-6226e8162e64\": \"d24b9a1b-8aaa-4c2c-981f-02c341f492d8\", \"a693e229-e24b-4d28-a21c-b1cd3f8a92b0\": \"d24b9a1b-8aaa-4c2c-981f-02c341f492d8\", \"801bedf0-0251-4b48-8168-47e945347438\": \"d24b9a1b-8aaa-4c2c-981f-02c341f492d8\", \"930baa2b-9f50-4c78-bb87-ebff92f6f618\": \"d24b9a1b-8aaa-4c2c-981f-02c341f492d8\", \"795ba408-0c4f-45f8-9f81-d86294b1c531\": \"cc4565c8-0b36-494d-b41a-2df7bf246bb4\", \"905f2e36-68a2-4c59-96f2-499a41b7feac\": \"cc4565c8-0b36-494d-b41a-2df7bf246bb4\", \"7237af19-a19c-461a-bdf2-d8853bbee804\": \"cc4565c8-0b36-494d-b41a-2df7bf246bb4\"}, \"doc_id_to_summary_id\": {\"c1e8c5b7-3294-41bb-8f3f-ae390b5f5387\": \"32fd9825-4422-4606-b587-46c1d2fe9f20\", \"35acfbaf-a1c5-41ef-81e9-a55058c41873\": \"d8c36da4-93f6-44be-a02d-3182b17eea55\", \"e3d5792f-fb41-4927-a27e-90bec4c037ee\": \"33f1feed-8e84-43f5-8f8b-595c16fda7ef\", \"ded38f6c-b012-4226-aae5-16c3e088bf41\": \"d24b9a1b-8aaa-4c2c-981f-02c341f492d8\", \"a0eb0e38-5794-4b8b-8874-444c12731121\": \"cc4565c8-0b36-494d-b41a-2df7bf246bb4\"}}"}}}
```

Comment:
Generally, if this file can be genereated by your code, we can remove this from the project.